BEIMA.Backend/UserFunctions/UpdateUser.cs

```diff
@@ -62,6 +62,10 @@ public static class UpdateUser
                 }
                 var originalUserRecord = BsonSerializer.Deserialize<User>(originalUserDoc);
 
+                // Enforce username case insensitivity
+                originalUserRecord.Username = originalUserRecord.Username.ToLower();
```

Comment:
I don't think this should be necessary, as we shouldn't be able to put lowercase usernames into the database in the first place.

Reply:
This is a sanity check, just in case there ever does exist a username with capital letters. Doesn't hurt to have.